src/ray/raylet/node_manager.cc

```diff
@@ -432,6 +433,15 @@ void NodeManager::HeartbeatAdded(gcs::AsyncGcsClient *client, const ClientID &cl
   }
 }
 
+void NodeManager::HeartbeatBatchAdded(gcs::AsyncGcsClient *client,
+                                      const UniqueID &publish_key,
+                                      const HeartbeatBatchTableDataT& heartbeat_batch) {
+  for (auto& heartbeat_data : heartbeat_batch.batch) {
+    ClientID client_id = ClientID::from_binary(heartbeat_data->client_id);
+    this->HeartbeatAdded(client, client_id, *heartbeat_data);
```

Comment:
Looking at the code, it doesn't seem like `client` is actually used in either this method or `HeartbeatAdded`. Can you remove the argument from both methods?

Reply:
Neither is `publish_key`. Should we remove that as well? I'm also confused about what that actually is.

Reply:
Oh hm yeah I just cherry-picked our commits from the benchmarks, I'll take a pass through.